kayenta-signalfx/src/main/java/com/netflix/kayenta/signalfx/security/SignalFxNamedAccountCredentials.java

```diff
@@ -34,8 +35,6 @@ public class SignalFxNamedAccountCredentials extends AccountCredentials<SignalFx
 
   @NotNull private String name;
 
-  @NotNull @Singular private List<Type> supportedTypes;
```

Comment:
This is the fix because https://github.com/spinnaker/kayenta/pull/926/files#diff-4b56b9f07abd19a8b5db2b64409e10bad9f639f0a1764e53934f97c03eb83caeR34 put `supportedTypes` in the base AccountCredentials class.

Reply:
Yep! I removed it from all the other accounts, apparently somehow missed signalfx.  I added a test, checked elsewhere couldn't find it elsewhere.  Got a test for it here, but long term if I can nuke this whole "copy the one into the other" code a lot of this'll go away.